models/perm/access/repo_permission.go

```diff
@@ -181,12 +211,6 @@ func GetUserRepoPermission(ctx context.Context, repo *repo_model.Repository, use
 		return perm, nil
 	}
 
-	if err := repo.LoadUnits(ctx); err != nil {
```

Comment:
Why we removed these lines?

Reply:
They are moved to above.

![image](https://github.com/go-gitea/gitea/assets/2114189/93b831a4-276d-4ef4-8805-1ccfcb39832a)